src/main/resources/static/gamelist.html

```diff
@@ -0,0 +1,134 @@
+    <p id="lastUpdated"></p>
+    <div id="stats-grid" class="stats-grid"></div>
+    <div id="games-container"></div>
+</div>
+
+<script>
+    function formatDateTime(dateString) {
+        const date = new Date(dateString);
+        return date.toLocaleString();
+    }
+
+    function formatDuration(startTimeString) {
+        const startTime = new Date(startTimeString);
+        const now = new Date();
+        const minutes = Math.floor((now - startTime) / (1000 * 60));
+
+        if (minutes < 60) {
+            return minutes + " min";
+        }
+        return Math.floor(minutes / 60) + "h " + (minutes % 60) + "m";
+    }
+
+    function createStatBox(label, value) {
+        return `
+            <div class='stat-box'>
+                <div class='stat-label'>${label}</div>
+                <div class='stat-value'>${value}</div>
+            </div>
+        `;
+    }
+
+    function createGameCard(game) {
+        const playersList = game.players.length === 0
+            ? "<div class='player-item'>No active players</div>"
+            : game.players.map(player => `
+                <div class='player-item'>
+                    <div>
+                        ${player.name}
+                        ${player.isHost ? "<span class='host-badge'>Host</span>" : ""}
+                    </div>
+                    <div>${formatDuration(player.joinTime)}</div>
+                </div>
+            `).join('');
+
+        return `
+            <div class='game-card'>
+                <div class='game-header'>
+                    <div>
+                        <h2 class='game-title'>
+                            ${game.name}
+                            <span class='game-type'>${game.version}</span>
+                        </h2>
+                        <div class='game-meta'>
+                            Game ID: ${game.id}
+                            • Started: ${formatDateTime(game.startTime)}
+                            • Max Players: ${game.maxPlayers}
+                        </div>
+                    </div>
+                </div>
+                <div class='player-list'>
+                    ${playersList}
+                </div>
+            </div>
+        `;
+    }
+
+    function updateMonitor() {
+        fetch('/games/api')
+            .then(response => response.json())
+            .then(data => {
+                // Update last updated time
+                document.getElementById('lastUpdated').textContent =
+                    'Last updated: ' + formatDateTime(data.timestamp);
+
+                // Update stats
+                const statsGrid = document.getElementById('stats-grid');
+                statsGrid.innerHTML =
+                    createStatBox('Active Games', data.stats.activeGames) +
+                    createStatBox('Total Players', data.stats.totalPlayers) +
+                    createStatBox('Avg. Players/Game', data.stats.averagePlayersPerGame.toFixed(1));

```

Comment:
> I did not realised, "Total players" is total players in game ("active" players, counting hosts)
> 
> Maybe it would be interesting to have "Total Players In Game" and "Total Players Online" (using the "personaConnection" entity), excluding hosts in both numbers is better I think, as they are not players, instead of the average players per game ?
> 
> We'll see with the community ;)

i'll make some changes, its not a problem

Reply:
Resolved by #52